src/extension.ts

```diff
@@ -107,6 +115,10 @@ export function activate(context: vscode.ExtensionContext) {
           const selection = await vscode.window.showWarningMessage(`${msg}. Powered by [Snyk](${registrationURL})`, StatusMessages.FULL_STACK_PROMPT_TEXT);
           if (selection === StatusMessages.FULL_STACK_PROMPT_TEXT) {
             vscode.commands.executeCommand(Commands.TRIGGER_FULL_STACK_ANALYSIS);
+            record('vulnerability_report_popup',{opened:true});

```

Comment:
Why not separate event names instead of using another argument(opened:true|false)?
vulnerability_report_popup_open
vulnerability_report_popup_ignore

Reply:
done